daprdocs/content/en/reference/components-reference/supported-name-resolution/nr-mdns.md

```diff
@@ -0,0 +1,27 @@
+---
+type: docs
+title: "mDNS Name Resolution Provider"
```

Comment:
Component, not provider

Reply:
See above comment. It's a name resolution provider component, just like it's a state store component or pubsub message broker spec.